.claude/hooks/block_inline_secrets.py

```
@@ -0,0 +1,63 @@
+
+# Strip the public demo JWT so it never trips detection.
+scan = cmd.replace(SUPABASE_DEMO_MARKER, "")
+
+for pattern, label in SECRET_PATTERNS:
+    if re.search(pattern, scan, re.IGNORECASE):
+        sys.stderr.write(
+            f"🚫 コマンドに認証情報が埋め込まれています: {label}\n"
+            f"   inline の秘密は settings.local.json に許可ルールとして永続化され、"
+            f"git 履歴への漏洩原因になります。\n"
+            f"   環境変数 もしくは シークレットマネージャ"
+            f"（1Password / Doppler / GitHub Secrets）経由で渡してください。\n"
```

Ruff (0.15.15)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)